ui/app/common/attributeTypes/views/attributeInformation.html

```diff
@@ -1,84 +1,117 @@
+                   ng-if="attribute.name != 'NICK_NAME'"
                    type="text" pattern-validate id="{{::attribute.name}}"
                    ng-model="targetModel[attribute.name]" data-item-type="personAttribute"
-                   ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-blur="::handleUpdate(attribute.name)">
+                   ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}"
+                   ng-blur="::handleUpdate(attribute.name)">
 
             <input data-ng-switch-default
-                   ng-if = "attribute.name == 'NICK_NAME'"
+                   ng-if="attribute.name == 'NICK_NAME'"
                    placeholder="{{ ::'REGISTRATION_LABEL_PATIENT_NICK_NAME' | translate}}"
                    type="text" pattern-validate id="{{::attribute.name}}"
                    ng-model="targetModel[attribute.name]" data-item-type="personAttribute"
-                   ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-blur="::handleUpdate(attribute.name)">
+                   ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}"
+                   ng-blur="::handleUpdate(attribute.name)">
         </span>
     </div>
-    <div class="left" data-ng-switch-when="org.openmrs.Concept">
+    <div class="left" data-ng-switch-when="java.lang.String" ng-if="attribute.name == 'occupation'">
+        <input type="text"
+               class="ui-autocomplete-input"
+               id="{{::attribute.name}}"
+               name="{{::attribute.name}}"
+               placeholder="Search Occupation"
+               ng-model="targetModel[attribute.name]"
+               ng-keyup="suggest(targetModel[attribute.name])"
+               ng-required="{{::attribute.required}}">
+        <ul class="ui-front ui-autocomplete ui-menu ui-widget ui-widget-content ui-corner-all" ng-if="showTag" ng-hide="hideList"
+            style="position:absolute; left:912px; top: 833px; width:191px">
+            <li class="ui-menu-item" role="presentation" ng-repeat="info in filterOcuppation"
+                ng-click="hideSuggestions(info)">
+                <a class="ui-corner-all" tabindex="-1">{{info}}</a>
+            </li>
+        </ul>
+    </div>
+
+    <div class="left" data-ng-switch-when="org.openmrs.Concept" ng-if="attribute.name != 'occupation'">
         <select id="{{::attribute.name}}" ng-model="targetModel[attribute.name].conceptUuid"
                 ng-options="a.conceptId as a.description for a in ::attribute.answers"
-                ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}" ng-change="appendConceptNameToModel(attribute); handleUpdate(attribute.name)">
+                ng-disabled="::isReadOnly(attribute.name)" ng-required="{{::attribute.required}}"
+                ng-change="appendConceptNameToModel(attribute); handleUpdate(attribute.name)">
```

Comment:
Looks like nothing is changed here except adding ng-change to the new line.Please keep it as it is